lib/transformer.js

```diff
@@ -414,6 +417,7 @@ Transformer.prototype.replace = function (rule, cb) {
     // 1.1 If no files were returned by grep, issue a warning and return
     if (files.length === 0) {
       self.addWarning(rule, 'Search did not return any results.');
+      self.script.addRule(rule);
```

Comment:
When the make the rule nothing may match, however after committing changes there may be a new match. So we need to save the rule in the script even though right now it doesn't do anything.